Security/src/CVE-2023-23397/CVE-2023-23397.ps1

```diff
@@ -61,6 +61,8 @@
     This optional parameter lets you ignore TLS certificate mismatch errors.
 .PARAMETER Credential
     This optional parameter lets you pass administrator credential object while running on Exchange Server on-premises.
+.PARAMETER TimeoutSeconds
```

Comment:
Can we add the parameter to the parameters table within the script documentation?

Reply:
Done.